code/game/turfs/turf_changing.dm

```diff
@@ -30,18 +30,23 @@
 		if(istype(below) && !isspaceturf(below))
 			N = /turf/simulated/open
 
-	var/old_air =       air
-	var/old_fire =      fire
-	var/old_above =     above
-	var/old_opacity =   opacity
-	var/old_density =   density
-	var/old_prev_type = prev_type
+	var/old_air =              air
+	var/old_fire =             fire
+	var/old_above =            above
+	var/old_opacity =          opacity
+	var/old_density =          density
+	var/old_corners =          corners
+	var/old_prev_type =        prev_type
+	var/old_lighting_overlay = lighting_overlay
 
 	changing_turf = TRUE
 
 	qdel(src)
 	var/turf/W = new N(src)
 
+	W.lighting_overlay = old_lighting_overlay
+	W.corners = old_corners
+
 	above = old_above
 	prev_type = old_prev_type
```

Comment:
Nothing to do with this PR, but these two lines looked like they got missed for `W.`-ification